dot/peerset/peerstate.go

```diff
@@ -258,17 +259,19 @@ func (ps *PeersState) addNoSlotNode(idx int, peerID peer.ID) {
 	}
```

Comment:
just a question: `numIn` and `numOut` can achieve negative values? @arijitAD

Reply:
numIn: number of connections where other nodes connected to us
numOut: number of connections where we connected to other nodes

don't think these values could be negative

Reply:
ok, should we add some sort of check to avoid these values being negatives?